wren-ai-service/src/pipelines/ask/indexing_pipeline.py

```diff
@@ -23,46 +23,112 @@
 DATASET_NAME = os.getenv("DATASET_NAME")
 
 
-class Indexing(BasicPipeline):
+@component
+class DocumentCleaner:
+    """
+    This component is used to clear all the documents in the specified document store(si).
+
+    """
+
+    def __init__(self, stores: List[DocumentStore]) -> None:
+        self._stores = stores
+        pass
```

Comment:
I am curious what's the purpose of "pass" here

Reply:
no more purpose. this is just that I use to put a pass before implementing a function. I can remove it. 👍